src/deprecated.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const deprecated = {
+  useFlowParser: config =>
+`  The ${"\"useFlowParser\""} option is deprecated. Use ${"\"parser\""} instead.
+
+  Prettier now treats your configuration as:
+  {
+    ${"\"parser\""}: ${config.seFlowParser ? "\"flow\"" : "\"babylon\""}
```

Comment:
Thanks for catching this! Posted a fix https://github.com/jlongster/prettier/pull/441